Sample/Vungle Sample App/BannerList.m

```diff
@@ -0,0 +1,139 @@
+//
+//  BannerList.m
+//  Vungle Sample App
+//
+//  Created by John Mai on 2/6/20.
+//  Copyright © 2020 Jordyn Chuhaloff. All rights reserved.
+//
+
+#import "BannerList.h"
+#import "Constants.h"
+#import "DisplayTableViewCell.h"
+
+@interface BannerList () {
+    CGFloat screenHeight;
+    CGFloat screenWidth;
+}
+@property (strong, nonatomic) IBOutlet UIView *view;
+@property const int totalSize;
+@property const int AD_PER_ITEM;
+@property NSMutableArray* adArr;
+@property (nonatomic, strong) VungleSDK *sdk;
+@end
+
+@implementation BannerList
+- (void)viewDidLoad {
+    [super viewDidLoad];
```

Comment:
in this method - is the sequence necessary? I'd change this up - I'd start vungle first or last. I'd also give some idea in comments why you're setting your total ad size or 40, or what the adPerItem value is used for (and why 4)?